doc/api/util.md

```diff
@@ -532,9 +536,10 @@ console.log(inspect(weakSet, { showHidden: true }));
 ```
 
 Please note that `util.inspect()` is a synchronous method that is mainly
-intended as a debugging tool. Some input values can have a significant
-performance overhead that can block the event loop. Use this function
-with care and never in a hot code path.
+intended as a debugging tool. Its maximum output length is limited to 128 MB and
+input values that result in output bigger than that will not be inspected fully.
```

Comment:
☝️ Since this is a heuristic is it approximately 128 mb or exactly?

Reply:
It should be detected by the very first entry that exceeds that limit (but only after being fully inspected at that level).

Reply:
And that is also not exact as the higher levels could already contain data that will be added on top of that. It is fairly precise if the individual properties on the object are not that big. If each of them is huge, the limit will not be as exact.

Reply:
Could the wording be tweaked to qualify it as "approximately 128 MB"?